### ISSUE TITLE
Jupyter notebook very slow at rendering Images.jl objects/crashes tab in Chrome

### ISSUE THREAD

**Issue by tlnagy** [JuliaImages/Images.jl#676]
Using `Images.jl` 0.11.3 and `IJulia.jl` 1.6.2 on OS X leads to extremely slow rendering of `AxisArray` objects. This came up when loading multidimensional images using [`OMETIFF.jl`](https://github.com/tlnagy/OMETIFF.jl) and then plotting in Jupyter notebook. Loading the same image using `imshow` from ImageView.jl was very quick. I've attached a video of the experience: 

https://youtu.be/LVL5oE3s_QA

The following code (if run in IJulia should reproduce the problem):

```julia
Pkg.checkout("OMETIFF", "tn/position-as-axis")
using FileIO
img = load("/Users/tamasnagy/Downloads/background_1_MMStack.ome.tif");
img[:, :, 1]
```
where the file is this one: https://github.com/tlnagy/OMETIFF.jl/blob/0ffadc4d6f8397ed529226132dd0af95c4e35bd4/test/testdata/singles/background_1_MMStack.ome.tif (~18 Mb file)

The last minute of the video is simply waiting on the IJulia/Images.jl display code I presume. I don't think `OMETIFF` is the culprit since it's returning a standard `AxisArrays` object.

**Comment by tlnagy** [comment]
The tab also reproducibly crashes in Chrome with the "Aw Snap" message.

**Comment by rdeits** [comment]
Hm, I remember seeing that sort of behavior when, for various reasons, the Images.jl display code wasn't being run, and instead it was falling back to Colors.jl, which displays a matrix of colors using a separate HTML `<rect>` foe each pixel. Could check if, when the image renders, it's actually rendered as an html `<img>` rather than a whole lot of `<rect>`s? (You can right-click on it and do "inspect" in Chrome).

**Comment by tlnagy** [comment]
So I did the inspect and it looks like it's making a SVG. Importing `Images.jl` leads to the same code making a PNG and fast performance. It looks like the `Colors.jl` default is the SVG option, which is horribly slow?

**Comment by tlnagy** [comment]
I guess the main way to make this run faster is to just import Images in my OME-TIFF reader so the correct `show` is used.

**Comment by rdeits** [comment]
Aha, yes. This is actually just a duplicate of https://github.com/JuliaImages/Images.jl/issues/619#event-1041419595 The fix in that case was to have ImageMagick.jl depend on (and import Images.jl). Can you do the same for OME-TIFF?

**Comment by Evizero** [comment]
Yes, displaying images in Jupyter without importing `Images.jl` is in general a bad idea.

**Comment by Evizero** [comment]
I ran into this before a couple of time

**Comment by tlnagy** [comment]
So I added `Images` to `OMETIFF`'s REQUIRE, which should fix this problem.

**Comment by rdeits** [comment]
You'll need to make sure you do `import Images` or `using Images` inside OMETIFF.jl as well.

**Comment by tlnagy** [comment]
Yeah, I saw that too. Should be fixed on master.
